addon/components/polaris-date-picker.js

```diff
@@ -81,6 +81,10 @@ export default Component.extend({
    */
   multiMonth: false,
 
+  hoverDate: null,
+
+  focusDate: null,
```

Comment:
`hoverDate` and `focusDate` are internal properties so should stay below the public properties